src/style.css

```diff
@@ -1 +1,80 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  /* stylelint-disable-next-line string-quotes */
+  font-family: 'Roboto', sans-serif;
+  font-size: 500;
```

Comment:
The font-size should have a unit, e.g., '16px' or '1em'. Probably you'd like to use font-weight?